themes/helm/assets/sass/docs-content.scss

```diff
@@ -1,3 +1,7 @@
+code span {
+  color: black !important;
```

Comment:
Hi! Would you mind explaining the reason behind this change?

Reply:
I didn't mean to commit this. This was a quick and dirty fix for #499 . Thanks for spotting this. I removed this from the commit.